packages/common/src/items/clearItemFeaturedImage.ts

```diff
@@ -0,0 +1,42 @@
+import { IUserRequestOptions } from "@esri/arcgis-rest-auth";
+import { removeItemResource } from "@esri/arcgis-rest-portal";
+import HubError from "../HubError";
+
+/**
+ * Given an item, and owner, remove a reource from the item
+ * @param id
+ * @param owner
+ * @param filename
+ * @param ro
+ */
+export async function clearItemFeaturedImage(
```

Comment:
sorry did not notice this earlier... but... this fn does nothing more than wrap `removeItemResource` + add some `HubError` sugar... To me, this suggests we don't really need it, and we can just use `removeItemResource`... Also it's still called `clearItemFeaturedImage` but does have any "Featured Image" related logic.